src/levanter/models/llama.py

```diff
@@ -259,11 +263,33 @@ def init(
         *,
         key,
         use_bias: bool = False,
+        use_mup: bool = False,
     ) -> "LlamaMlp":
         k_fc, k_up_proj, k_down_proj = jrandom.split(key, 3)
-        gate_proj = hnn.Linear.init(Out=Mlp, In=Embed, key=k_fc, use_bias=use_bias, out_first=True)
-        up_proj = hnn.Linear.init(Out=Mlp, In=Embed, key=k_up_proj, use_bias=use_bias, out_first=True)
-        down_proj = hnn.Linear.init(Out=Embed, In=Mlp, key=k_down_proj, use_bias=use_bias, out_first=True)
+        gate_proj = hnn.Linear.init(
+            Out=Mlp,
+            In=Embed,
+            key=k_fc,
+            use_bias=use_bias,
+            out_first=True,
+            reparam_cls=mup.HiddenLinearMup if use_mup else mup.LinearStandardParam,
```

Comment:
kinda wonder if we should add aliases like

```
def hidden(use_mup=True): return mup.HiddenLinearMup if use_mup else mup.LinearStandardParam
```

and then do mup.hidden() but eh

Reply:
Yeah, good call. Will add on the Haliax PR first and then move here!